samples/client/petstore/cpp-qt5/client/OAIUserApi.h

```diff
@@ -15,7 +15,7 @@
 
 #include "OAIHttpRequest.h"
 
-#include "OAIUser.h"
+#include "OAIOAIUser.h"
```

Comment:
I wonder what happened here?

Reply:
I could reproduce this stuff in the latest master branch as well. Stepped through the 'cpp-qt5' generator and it seems their 'PREFIX=OAI' is applied multiple times. I would like to create an issue for that.